### ISSUE TITLE
how many epoches did you use to evaluate a dataset ? 

### ISSUE THREAD

**Issue by hurricane2018** [shamangary/DeepCD#3]
The test accuracy is always change. Did you get the average precision of the test data ?

I mean, If we train 10 epoch, we can get a precision. If we train 20 epoches, we can get another result. How did you asses the accuracy of a dataset?

**Comment by shamangary** [comment]
This repository is only for the demo of the training. You may reduce the learning rate to get more stable result.

**Comment by hurricane2018** [comment]
Yes,  I mean we can get an accuracy when we train 10 epoches. Then, we can get an other accuracy when we train 10 epoches again? 

I mean how did u get your results in your paper? Did you take some average results of your trial ?

Jie

**Comment by shamangary** [comment]
Usually we only take the final result since the previous epoch may not converge yet. Therefore taking average result on No.10 and the latter epoch is not reasonable. The rest of the setting is stated in the paper. Thanks !
